### ISSUE TITLE
3d dense UNET for vertebrae segmentation

### ISSUE THREAD

**Issue by reygz** [mrkolarik/3D-brain-segmentation#9]
Hi, the spineweb dataset2 contains vertebrae level labels for vertebrae segmentation. Any suggestions on how to modify/extend 3d dense UNET for vertebrae segmentation?

**Comment by mrkolarik** [comment]
Hi,

well there are multiple possible routes how to proceed, but If you want to implement the vertebrae segmentation and classification as an end-to-end task using single neural network, then I would treat this problem as a multi class segmentation.

You would have to encode the data properly and then change the output layer of the net to softmax activation with as many channels as there are classes. This article describes something similar:
https://naomi-fridman.medium.com/multi-class-image-segmentation-a5cc671e647a

Hope this helps :)

Take care,

Martin
